Python/Product/TestAdapter.Executor/Config/RunSettingsUtil.cs

```diff
@@ -8,12 +8,13 @@
 
 namespace Microsoft.PythonTools.TestAdapter.Config {
     public class RunSettingsUtil {
-        public static Dictionary<string, PythonProjectSettings> GetSourceToProjSettings(IRunSettings settings) {
+        public static Dictionary<string, PythonProjectSettings> GetSourceToProjSettings(IRunSettings settings, TestFrameworkType filterType) {
```

Comment:
now that we have 2 separate classes being triggered for discovery we can do less work by skipping over projects that don't have the test framework we are looking for